modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -758,15 +758,15 @@ public void prepareRemoval(KillEvent killEvent)
     @Override
     public void postRemoval(KillEvent ce)
     {
-        _log.info("CellAdapter : prepareRemoval : waiting for gate to open");
+        logger.info("CellAdapter : prepareRemoval : waiting for gate to open");
```

Comment:
true. We just thought it might be more efficient to do it both at once. shall we do it again/ seperate the two changes?